src/unix/linux_like/linux/musl/mod.rs

```diff
@@ -756,6 +745,73 @@ extern "C" {
     pub fn malloc_usable_size(ptr: *mut ::c_void) -> ::size_t;
 }
 
+// Musl's standard entrypoints are already LFS64 compatible, historically the library aliased
+// these together in header files (as `#define`s) _and_ in the library with weak symbol aliases.
+//
+// Since <version> these aliases were removed from the library (both in the API and the ABI) so we
+// alias them here to keep the crate API stable.
+#[allow(dead_code)]
+fn check_type_aliases(
+    dirent: ::dirent,
+    ino: ::ino_t,
+    flock: ::flock,
+    off: ::off_t,
+    pos: ::fpos_t,
+    rlimit: ::rlimit,
+    stat: ::stat,
+    statfs: ::statfs,
+    statvfs: ::statvfs,
+) {
+    let _dirent: ::dirent64 = dirent;
+    let _ino: ::ino64_t = ino;
+    let _flock: ::flock64 = flock;
+    let _off: ::off64_t = off;
+    let _pos: ::fpos64_t = pos;
+    let _rlimit: ::rlimit64 = rlimit;
+    let _stat: ::stat64 = stat;
+    let _statfs: ::statfs64 = statfs;
+    let _statvfs: ::statvfs64 = statvfs;
+}
```

Comment:
This could be moved to test, I guess?

Reply:
Could be, though this is merely a compile-time check that the interchangeable types are aliases (rather than identically defined separate types) it's not a test in the classical sense.

An alternative might be to `#[cfg(test)]` this function?